cdap-proto/src/main/java/co/cask/cdap/proto/codec/ScheduleSpecificationCodec.java

```diff
@@ -14,7 +14,7 @@
  * the License.
  */
 
-package co.cask.cdap.internal.app;
+package co.cask.cdap.proto.codec;
```

Comment:
If we move one codec there, do we want to move all ``*Codec`` classes there? Please check with @chtyim or @anew